repository/c.json

```diff
@@ -3118,6 +3118,17 @@
 				}
 			]
 		},
+		{
+			"name": "CompetitiveProgrammingParser",
+			"details": "https://github.com/codeuniverse101/CompetitiveProgrammingParser",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"platforms": ["*"],
```

Comment:
If you support all platforms, please just remove the key here.

maybe a push on the PR also kicks the review-bot into action 🤞

Reply:
Done